inductiva/api/methods.py

```diff
@@ -256,16 +256,46 @@ def blocking_task_context(api_instance: TasksApi, task_id):
         signal.signal(signal.SIGINT, original_sig)
 
 
-def submit_task(api_instance, task_request, params, type_annotations):
+def log_task_info(task_id, method_name, params, resource_pool):
+    """Logging the main components of a task submission."""
+
+    logging.info("Task Information:")
+    logging.info("> ID: %s", task_id)
+    logging.info("> Method: %s", method_name.split(".")[-1])
+    logging.info("> Local input directory: %s", params["sim_dir"])
```

Comment:
Pet peeve
```suggestion
    logging.info("Task Information       :")
    logging.info("> ID                   : %s", task_id)
    logging.info("> Method               : %s", method_name.split(".")[-1])
    logging.info("> Local input directory: %s", params["sim_dir"])
```

Reply:
I am not sure if I comply with your pet peeve, mostly because of the dots being super spaced, but let me see what I can do!